config/eslintrc_react.js

```diff
@@ -40,6 +41,7 @@ module.exports = {
         'react/no-is-mounted': 2,
         'react/no-multi-comp': 0, // Enable to define a multiple component to a single file.
         'react/no-render-return-value': 2,
+        'react/no-redundant-should-component-update': 0, // we'll reconsider this option when we begin to use `PureComponent`.
```

Comment:
Ah, I've had misunderstanding. I had understood the behavior of `ignorePureComponents` oppositly.

If `ignorePureComponents` is true:

- we prefer stateless component
- and we can use PureComponent
- but PureComponent without its props or context cause warning

Is it right?